conans/test/integration/package_info_test.py

```diff
@@ -235,3 +237,169 @@ def package_info(self):
         client.run("create conanfile.py dep/1.0@us/ch", assert_error=True)
         self.assertIn("dep/1.0@us/ch package_info(): self.cpp_info.components cannot be used "
                       "with self.cpp_info configs (release/debug/...) at the same time", client.out)
+
+    def package_info_wrong_cpp_info_test(self):
```

Comment:
This was already tested, wasn't it?  Not the same fields, but it is testing the same functionality

```
def package_info(self):
                    self.cpp_info.defines.append("defint")
                    self.cpp_info.components["int1"].libs.append("libint1")	                   
```